<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="ImageView" name="inputImageView"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="hiddenDataSize">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="inputImageInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="buttonImage">
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonEncrypt">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Encrypt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDecrypt">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Decrypt</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">imageview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonImage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onInputImageClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>-21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEncrypt</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onEncryptClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>-96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDecrypt</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDecryptClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>-82</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onInputImageClick()</slot>
  <slot>onEncryptClick()</slot>
  <slot>onDecryptClick()</slot>
 </slots>
</ui>
